### DISCUSSION TITLE
Webhook notifications - Can't receive

### DISCUSSION THREAD

**Discussion by wafgomes** [netflie/whatsapp-cloud-api#172] [General]
I'm stuck on cofigure how to receive notification from webhook.

The verification from `$webhook->verify($_GET, TOKEN)` is ok but I can't get anything from webhook via POST. already subscribed on all fields

trying to get from this route, witch is configured on 

`Route::post('/whatsapp/webhook', [WebhookController::class, 'index']);`

running on development mode

**Comment by wafgomes** [comment]
I'm ashamed to say that I spent two days to realize that I was returning a wrong response code 😭
everything is working fine

**Reply by rjuarez1** [reply]
Hi!!! Could you help me to implement webhook please?
